language-server/modules/langserver-compiler/src/test/java/org/ballerinalang/langserver/compiler/workspace/repository/WorkspacePackageRepositoryTest.java

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.ballerinalang.langserver.compiler.workspace.repository;
+
+import org.ballerinalang.langserver.compiler.LSCompilerUtil;
+import org.ballerinalang.langserver.compiler.LSPackageCache;
+import org.ballerinalang.langserver.compiler.common.LSDocument;
+import org.ballerinalang.langserver.compiler.workspace.ExtendedWorkspaceDocumentManagerImpl;
+import org.ballerinalang.langserver.compiler.workspace.WorkspaceDocumentManagerImpl;
+import org.ballerinalang.model.elements.PackageID;
+import org.ballerinalang.repository.PackageRepository;
+import org.ballerinalang.repository.PackageSource;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.wso2.ballerinalang.compiler.Compiler;
+import org.wso2.ballerinalang.compiler.tree.BLangPackage;
+import org.wso2.ballerinalang.compiler.util.CompilerContext;
+import org.wso2.ballerinalang.compiler.util.Name;
+import org.wso2.ballerinalang.compiler.util.Names;
+import org.wso2.ballerinalang.compiler.util.diagnotic.BLangDiagnosticLog;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Tests for WorkspacePackageRepository.
```

Comment:
Let's add the version

Reply:
Fixed in [ecaf52d](https://github.com/ballerina-platform/ballerina-lang/pull/10194/commits/ecaf52d98daa0a19f68c882272fbae7284876b24)